### ISSUE TITLE
App feature request: Show stored app data as visualization of contact events

### ISSUE THREAD

**Issue by phlibi** [DP-3T/documents#328]
Hi everyone,

I would love to see a feature in the app which shows its recorded data. As far as I understand its workings, it keeps a record of all exposures (infected or not) over the last 14 days. Having this visualized would (in my opinion) give at least two advantages:

- Users can quickly check if the proximity tracing is actually working, or if some system settings (e.g. app permissions) prevent it. It could also increase app trust of a user.

- Users get a feeling about how well they follow the "social distancing" recommendations.

The visualization could for example involve two bar graphs. The X axis would show the past time, resolved to, say, 1h. The Y axis of the first graph could show the total exposure time to other app users. The Y axis of the second graph could show the number of distinct contacts during the same time slots.

These are just suggestions, there will certainly be better ideas around. In case the SwissCovid app already includes such a feature, please tell me how to find it.

btw: I posted this issue to the documents project, as there are the most issues already present. Please comment if there is a better place for general feature requests.

Thanks and best regards, Philipp

**Comment by phlibi** [comment]
Possibly related issues: #40 #220

**Comment by noci2012** [comment]
It can also be used by law-enforcement to assess if you are a risk person or not. Or even set a limit on the amount of contacts one may have per day.
Or for door kepers:  Too many contacts ==> no access to this ... building.
obviously for the safekeeping of others in the building.

An indication of 0, 1, 2 many might be ok, an count of how many even an approximation can be a risk factor.

**Comment by phlibi** [comment]
I agree that such a feature could make it somewhat easier for the app to be abused. On the other hand, if users are in a situation where they are forced to show information of some app, then they can also be forced to install a specially crafted version (or even another app) for this purpose. This is why I doubt that the feature would make the app better or worse from a privacy point of view. It would be important that it does not collect more data of course, but only shows what it collects anyway.
